src/System.Management.Automation/FormatAndOutput/common/BaseOutputtingCommand.cs

```diff
@@ -987,8 +987,25 @@ internal override void Initialize()
                     columnWidthsHint = tableHint.columnWidths;
                 }
 
-                _consoleHeight = GetConsoleWindowHeight(this.InnerCommand._lo.RowNumber);
-                _consoleWidth = GetConsoleWindowWidth(this.InnerCommand._lo.ColumnNumber);
+                try
+                {
+                    _consoleHeight = GetConsoleWindowHeight(this.InnerCommand._lo.RowNumber);
+                }
+                catch (Exception)
```

Comment:
Is the `Exception` needed here? If you're catching the most general kind of exception and not doing anything with it, it should be possible to just omit the expression